docs/snippets/ov_python_exclusives.py

```diff
@@ -131,3 +131,39 @@ def f(request, userdata):
 
 assert all(data_done)
 #! [asyncinferqueue_set_callback]
+
+#! [releasing_gil]
+import openvino.runtime as ov
+import cv2
```

Comment:
consider using `import cv2 as cv` + `cv.` (`cv2` is going to be deprecated)

Reply:
Done.